homeworks/yana.haivoronska_Yana_Haivoronska/Game Project/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href="style.css">
+
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <title>About me</title>
+    </head>
+    <body>
+        <main>
+            <h1>About me</h1>
+            <p>
+                Hi! My name is Yana Haivoronska, and I'm a Junior Frontend
+                Developer. I am already familiar with main Web Technologies like
+                HTML, CSS, JavaScript and Git version control system.
+            </p>
+            <p>
+                This page was developed during the course
+                <a
+                    style="color: red" href="https://www.mastersacademy.education/frontend-for-beginners-it"
```

Comment:
It should be better to apply styles to class. Please avoid usage of inline styles